README.md

```diff
@@ -8,7 +8,7 @@
 
 Repo for you to raise a Pull Request for practice.
 
-**Just add your name to the alphabetical list and optionally, a link to your GitHub account**
+** Just add your name to the alphabetical list and optionally, a link to your GitHub account**
```

Comment:
This removes the bold text from this line.

Is this intentional?

Reply:
No. It's auto formatting from my code editor.